python/ray/tests/kuberay/test_autoscaling_config.py

```diff
@@ -35,6 +37,20 @@ def get_basic_ray_cr() -> dict:
     )
     gpu_group["maxReplicas"] = 200
     config["spec"]["workerGroupSpecs"].append(gpu_group)
+    tpu_group = copy.deepcopy(config["spec"]["workerGroupSpecs"][0])
```

Comment:
Do we need to set `numOfHosts`?

Reply:
Done in [ec92b79](https://github.com/ray-project/ray/pull/45105/commits/ec92b791e9d706b0986c9132cd0d82855346b5cd).